api/instance/v1/server_utils.go

```diff
@@ -236,3 +237,61 @@ func (s *API) SetServerUserData(req *SetServerUserDataRequest, opts ...scw.Reque
 
 	return nil
 }
+
+// GetAllServerUserDataRequest is used by GetAllServerUserData method.
+type GetAllServerUserDataRequest struct {
+	Zone     utils.Zone `json:"-"`
+	ServerID string     `json:"-"`
+}
+
+// GetAllServerUserDataResponse is used by GetAllServerUserData method.
+type GetAllServerUserDataResponse struct {
+	UserData map[string]io.Reader `json:"-"`
+}
+
+// GetAllServerUserData get all server user data.
```

Comment:
user data of all servers? or all user data of one server?

maybe user more words in your comment 😜